lib/createContainer/createContainer.js

```diff
@@ -39,11 +39,14 @@ function createContainer(InnerComponent, config) {
         observer.dispose();
       }
     },
+    componentWillReceiveProps(nextProps) {
+      config.props = nextProps;
+    },
     getInitialState() {
       config = _.defaults(config, {
         // Have a default implementation of done so it can be
         // called from other handlers
-        done: (results) => {
+        done: function (results) {
```

Comment:
But it isn't. With ES6 fat arrow when you call done.call(config, results) it does not use the config as this because when getInitialState is called the function is bound to the component.